### ISSUE TITLE
Cannot build

### ISSUE THREAD

**Issue by obeone** [dpa99c/cordova-custom-config#32]
I haven't build since 14 days, but today, I don't know why, nothing happens with a cordova build (android or ios) : 
```
$ cordova build android
cordova-custom-config: Installing plugin dependencies...
cordova-custom-config: Skipping auto-restore of config file backup(s) due to config.xml preference
$
```

I have the problem on 3 different computers (each was able to build 14 days ago), and if I remove cordova-custom-config plugin, it works again. Maybe a problem with the new cordova-android or cordova-ios ?

**Comment by dpa99c** [comment]
I've completely reworked the dependency resolution script in the last couple of releases, so that's the difference from 14 days ago. I was attempting to fix race conditions and thought I'd fixed it, however...

I will look into it and publish a new version if I can find and fix the bug.

Bear with me...

**Comment by dpa99c** [comment]
Yep, found+fixed the bug. Only happened if `<preference name="cordova-custom-config-autorestore" value="false" />`, but the deferred promise was not being resolved, hence the build process never progressed past the hook. Should now be fixed in v1.2.4
